File: Strings/Notes.c
// Sasha Rinitra, Strings notes C
#include <stdio.h>
#include <string.h>

char name[20];

int main(void) {
    //printf("Please tell me your full name:\n");
    //scanf("%s", name); doesn't work or full name, only puts first name
    //fgets(name, 20, stdin);
    //printf("Hello %s, Welcome to my program", name);

    //char sentence[] = "The quick brown fox jumps over the lazy dog";
    //printf("%c\n", sentence[16]); // to print just 'f'
    //printf("%lu\n", sizeof(sentence)); //said 44, because it gives indexes
    //printf("%d\n", strien(sentence)); //says 43

    char one[] = "Hello ";
    char two[] = "World!";
    char three[] = "This is my program. ";
    two[5] = '?';
    printf("%s\n", one); //before concatination
    strcat(one, two);
    printf("%s\n", one);  //after concatination
    strcat(three, one);
    printf("%s", three);
  return 0;
}

File: Strings/Notes.py
#Sasha Rinitra, Strings Notes

# Strings are words in programming, specifically data types "" ''

print("Hello WRLD")
print('Hello WRLD')

# it will show error if ==> print('Hello to Sasha's World')

# People are stupid
name = input("What is your name?\n").strip().lower()
name = input("What is your name?\n").strip().upper()
name = input("What is your name?\n").strip().capitalize()

print(f"Hello {name}, Welcome to my Program!")

# Stupid proofing
name = input("What is your (first) name?\n").strip().capitalize()
print(f"Hello {name}, Welcome to my Program!")

# Syntax Error: error in te structure or punctuation
# Logic Error: sometimes 8+2 = 82
# In python, input is a string
name = input("Give me a number\n").strip().capitalize()
print(name + "2") #this doesn't work

# Concantination is when 2 strings are put together with a + symbol
# Debugging is finding errors in the system and fixing it
# Bug is a error in the system

name = input("Give me a number\n").strip().capitalize()
num = 5*2+name
print(num)
print(name + "2") #type error

# index number

sentence = "The quick brown fox jumps over the lazy dog"
print(sentence.find("fox"))

word = sentence.find("fox")

